core/src/main/java/com/novoda/merlin/MerlinsBeard.java

```diff
@@ -10,7 +11,8 @@
  */
 public class MerlinsBeard {
 
-    private ConnectivityManager connectivityManager;
+    private final ConnectivityManager connectivityManager;
+    private final TelephonyManager telephonyManager;
```

Comment:
is another permission needed to use the `TelephonyManager` if so it needs to be docummented :exclamation: